content/v0.75.2.md

```diff
@@ -0,0 +1,129 @@
+acceleration and sensitivity settings. This works in
+fullscreen or when the mouse is captured in window mode.
+
+Option can be changed in the sdl section of a configuration file:
+
+```
+[sdl]
+raw_mouse_input = true
+```
+
+This option is disabled by default, but some games (e.g. FPS titles,
+like DOOM or Dark Forces) will work better with raw input enabled
+(especially when used with modern, very sensitive gaming mouses).
+
+See: [#636](https://github.com/dosbox-staging/dosbox-staging/pull/636)
+
+
+## 3. Bugfixes
+
+- Bundle C++ Runtime libraries with Windows builds for users
+  on Windows 7, 8.x, and those who disabled Windows 10 update mechanism.
+  ([#675](https://github.com/dosbox-staging/dosbox-staging/pull/675))
+- Reimplement CLS command to prevent text mode reset.
+  ([#639](https://github.com/dosbox-staging/dosbox-staging/pull/639))
+- Fix whitespace handling in modem ATD command.
+  ([#652](https://github.com/dosbox-staging/dosbox-staging/pull/652))
+- Fix boot failure when using HDD image.
+  ([#522](https://github.com/dosbox-staging/dosbox-staging/pull/522))
+
+
+## 4. Notes for Packagers
+
+This release includes numerous fixes affecting our build system and other
+changes relevant for packaging (contributed by Gentoo packagers, and people
+preparing Debian, Fedora, and Flatpak packages) - prompt upgrade is recommended.
+
+If you're packaging DOSBox Staging for your OS, we have few tips:
+
+- We strongly suggest to keep our default binary name: `dosbox`.
+  DOSBox&nbsp;Staging is a drop-in replacement for older DOSBox versions - there
+  are programs depending on `dosbox` binary name (e.g. Wine or GUI frontends).
+  It's also the default name of our man page and appears in the documentation.
+- You don't need to generate your own desktop entry, appstream metadata
+  (metainfo.xml), nor package description - relevant files are in
+  `contrib/linux/` directory.
+- We provide icon in scalable format and icons optimized for small sizes -
+  look into `contrib/icons/` directory.
+- During the configuration step, remember to add <code>-D[NDEBUG](https://en.cppreference.com/w/c/error/assert)</code>
+  to preprocessor flags (it's important!), and we recommend using `-O3` in both
+  C and C++ compiler flags, for example:
+
+      ./configure CPPFLAGS="-DNDEBUG" CFLAGS="-O3" CXXFLAGS="-O3"
+
+- Our testing indicates, that GCC generates faster binaries than Clang, but
+  use whatever compiler is more appropriate for your OS.
+- The license is "GNU GPL version 2.0 **or later**" (SPDX: `GPL-2.0-or-later`).
+  GitHub does not show "or later", but our packages should show the proper
+  license info.
+
+To all our package maintainers: Thank you! :)
+
+
+## 5. Contributors
+
+**Maintainers:**
+
+Patryk Obara, kcgen
+
+**Contributors:**
+
+- Bernard Cafarelli
+- David Heidelberg
+- Feignint
+- NicknineTheEagle
+- WengierWu
+
+## 6. Changelog
+
+You can read detailed changelog via Git, e.g. using the following command:
```

Comment:
The detailed per-commit changelog can be read using the following Git command:

Reply:
After a bit of consideration… I decided to drop this section altogether. It's a leftover from a time when we had a long list of commits in the release notes. Technical and interested users probably know how to use git log on their own; for non-technical users it's pointless distraction.